simapp/upgrades.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // UpgradeName defines the on-chain upgrade name for the sample SimApp upgrade
-// from v047 to v050.
+// from v0.50.x to v0.51.x
```

Comment:
This is solely so that we have a go diff and trigger the action. I had to find a small useful change to add